lib/vdsm/storage/lvmdevices.py

```diff
@@ -98,7 +99,13 @@ def _devices_file_exists():
     Returns True if default lvm devices file exists. If the file doesn't
     exists, lvm disables whole devices file functionality.
     """
-    return os.path.exists(_LVM_SYSTEM_DEVICES_PATH)
+    try:
+        conf_devices_file = lvmconf.configured_value("devices/devicesfile")
```

Comment:
Minor issue - it would be little bit nicer if we keep the name of the option:

    devicesfile = lvmconf.configured_value("devices/devicesfile")

    path = os.path.join(_LVM_SYSTEM_DEVICES_DIR, devicesfile)

Reply:
done